api/db-cdr/generate-cdr/cloudsql-import-bucket.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Imports all sql and csv files in the bucket to the database.
+# IF --create-dump-file is specified, it is ran first
+# Table names for csvs are determined from the filename.
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+CREATE_DUMP_FILE=
+
+# get options
+USAGE="./generate-clousql-cdr/cloudsql-import-bucket.sh --project <PROJECT> --instance <INSTANCE> --bucket <BUCKET>"
```

Comment:
Could you include usage for --create-dump-file? Looks like you're expecting that to be a string - consider also renaming it since it sound s like a boolean flag currently.

Reply:
Ah, I was very mislead by the name I guess. I thought this was actually going to create a dump file, now the functionality makes sense.

I'd say something like `--create-db-from-sql-dump` or `--init-db-from-sql-dump` to make the workflow clearer.